molgenis-data-mapper/src/main/resources/js/attribute-mapping.js

```diff
@@ -274,7 +355,10 @@
 			// update algorithm
 			algorithm = editor.getSession().getValue();
 
-			// update result
+			// validate mapping
+			validateAttrMapping(algorithm);
```

Comment:
I don't understand your comment. The validation is not triggered when displaying the current algorithm by design. Do you mean this behavior should be different?